ios/file_picker/Sources/file_picker/FilePickerPlugin.m

```diff
@@ -528,19 +531,37 @@ -(void)picker:(PHPickerViewController *)picker didFinishPicking:(NSArray<PHPicke
     __block NSInteger completedCount = 0;
     NSInteger totalCount = results.count;
 
+    bool isImageSelection = self.type == IMAGE;
+    bool isMediaSelection = self.type == MEDIA;
     for (NSInteger index = 0; index < results.count; ++index) {
         dispatch_group_enter(_group);
         PHPickerResult * result = [results objectAtIndex:index];
         
         dispatch_async(processQueue, ^{
             @autoreleasepool {
+            if (isMediaSelection) {
+                if (![result.itemProvider hasItemConformingToTypeIdentifier:@"public.image"] &&
```

Comment:
Can we have these type identifiers as local constants? Or is there a constant on Objective-C to use here?

Reply:
Realistically I went through some of the prior attempts and by trial and error figured these two cases that cover image, video and mixed. The obj-c syntax is quite cryptic to me.

Reply:
I tried to use `UTTypeMovie` and `UTTypeImage` but started getting exceptions so I guess I would leave it as is?

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[_UTCoreType length]: unrecognized selector sent to instance 0x1eb9eabd0'
```

Reply:
Yeah, leaving it s is makes sense then.